recipes/xege/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+cmake_minimum_required(VERSION 3.1)
+project(PackageTest CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(example example.cpp)
+target_link_libraries(example ${CONAN_LIBS})
+
+if(MSYS OR MINGW)
+    target_link_libraries(example -static-libgcc -static-libstdc++)
+endif()
```

Comment:
why?

Reply:
When I'm testing this under 32bit MinGW-w64, I got some unexpected linkage error to the standard library. So I just use static linkage to prevent them.

This problem doesn't appear in 64bit MinGW, and I don't know why it will happen.

Reply:
Seems that these options are enabled in default for those IDE using 32bit MinGW:
https://github.com/Embarcadero/Dev-Cpp/blob/master/Source/devCFG.pas#L1297

Reply:
it's odd, but isolated in test package.